scripts/player.gd

```diff
@@ -1,7 +1,7 @@
 extends KinematicBody2D
 var velocity:Vector2 = Vector2(0,0)
```

Comment:
can you also fix the spacing here between the type and variable name

var velocity : Vector2